feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/SearchScreenViewModel.kt

```diff
@@ -71,8 +71,8 @@ class SearchScreenViewModel @Inject constructor(
             SearchScreenUiState.Empty(
                 searchQuery = SearchQuery(searchQuery),
                 searchFilterDayUiState = searchFilterDayUiState(filters.days),
-                searchFilterCategoryUiState = searchFilterCategoryUiState(filters.categories),
-                searchFilterSessionTypeUiState = searchFilterSessionTypeUiState(filters.sessionTypes),
+                searchFilterCategoryUiState = searchFilterCategoryUiState(filters.categories, sessions.categories),
+                searchFilterSessionTypeUiState = searchFilterSessionTypeUiState(filters.sessionTypes, sessions.sessionTypes),
```

Comment:
Thanks! It's better but, if there is a UiState that has all FilterSessionTypeUiStates, we could make this easy 👀